api/v1/swagger.yaml

```diff
@@ -3094,6 +3094,17 @@ paths:
             Filter by parentAbsolutePath; asterisk (*) wildcards ok.
           required: false
           type: string
+        -
+          name: tags
+          in: query
+          items:
+            type: string
+            maxLength: 60
```

Comment:
Each tag length is max 60 char long, as restricted by model.
Also, the changes in this PR only handles includes at sql level like `v1/subjects?tags=mytag`. 
Do we need excludes here as well? I assumed the story was about includes only to get trust data working. I can create another PR if we need excludes as well.